src/Components/NavModal/SocialLogin/SocialLogin.js

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+
+const { Kakao } = window;
+function SocialLogin() {
+  const kakaoLoginHandle = () => {
+    Kakao.Auth.login({
+      success: function (authObj) {
+        fetch('http://localhost:3000/oauth', {
+          method: 'POST',
+          header: JSON.stringify({
+            access_token: authObj.access_token,
+          }),
+        })
+          .then(res => res.json())
+          .then(res => {
+            localStorage.setItem('Wexpert_token', res.conconsong_token);
+            if (localStorage.setItem) {
+              alert('환영합니다!');
+            }
+          });
+      },
+      fail: function (err) {
+        alert(JSON.stringify(err));
+      },
+    });
+  };
+
+  return (
+    <LoginContainer>
+      <LoginText>
+        클릭 한 번으로
+        <br /> 당신의 전문가에게
+      </LoginText>
+      <LoginBtn kakao onClick={kakaoLoginHandle}>
+        <i className="fas fa-comment"></i>
+        카카오 로그인
+      </LoginBtn>
+    </LoginContainer>
+  );
+}
+
+const LoginContainer = styled.div`
+  display: flex;
+  position: absolute;
+  left: -80px;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  border: 1px solid #b5b5b5;
+`;
+
+const LoginText = styled.p`
+  margin-top: 10px;
+  font-size: 12px;
+`;
+
+const LoginBtn = styled.button`
+  width: 100px;
+  border: none;
+  margin: 10px 10px 20px 10px;
+  background-color: ${props => (props.kakao ? '#FEE500' : 'black')};
+  color: ${props => (props.kakao ? 'black' : 'white')};
```

Comment:
- kakao 라는 props 는 항상 부여되기 때문에, 항상 '#FEE500' 과 'black' 이 나올 것 같습니다.
- 따라서 여기에 로직을 부여하는게 의미가 있을까유?
- 이후에 추가적으로 구현될 사항이라면 오케이~
```suggestion
  background-color: '#FEE500';
  color: 'black';
```

Reply:
변경했습니다!